src/dom.ts

```diff
@@ -126,6 +127,48 @@ export function containsClass(element: HTMLElement, className: string): boolean
 	return targetClass.indexOf(' ' + className + ' ') > -1;
 }
 
+/*
+ * Creates an Element
+ *
+ * @param tagName Type of element to create
+ * @param kwArgs An object containing attributes to add to the Element. If an "attributes"
+ * property is present, each member of this sub-object will be added to the Element via element.setAttribute
+ * @param children An array of either Nodes or strings, the latter of which will be converted to Nodes via dom.fromString
+ *
+ * @example
+ * var div = dom.create('div', {'data-index': 0, attributes: {'class': 'loaded'}});
```

Comment:
A data-attribute would be in `attributes`, not the properties.  Might want to use that as the `attributes` example and think of something else for properties (could use `className`...)